go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/mongodb/mongodb-atlas-cli/atlascli
 
-go 1.22
+go 1.22.1
```

Comment:
why? and I guess is because you have it as a whole line https://github.com/mongodb/mongodb-atlas-kubernetes/blob/c4c22cf6fb0d0af8fb07d75323e40662d458b2bc/go.mod#L3 ?

Reply:
I did not change this myself. I think go mod tidy did.

What is the significance? doe the CLI project prefer one over the other?

Reply:
I see go 1.22 is less restrictive. I can try to change it back.

Reply:
Makes sense. Probably best to force just `1.22` but I guess I had to change the toolchain line as well.

Reply:
I make the edit then run go mod tidy and it changes it back...

Reply:
@gssbzn can you get away with changing this and run `go mod tidy` and not have it undone?

If you succeed, what version of go do you have? any special config? (I am using `go 1.22.1` at the moment, nothing special aside AFAIK)

Reply:
not a blocker, like I said just trying to understand in the general go ecosystem way since it may be a lot of to ask for consumers of libs to update their minimum all the time a lib decides to update theirs, feels like. abad experience but for now no issue